### DISCUSSION TITLE
Support the project folder parameter for monaco download

### DISCUSSION THREAD

**Discussion by robertjahn** [Dynatrace/dynatrace-configuration-as-code#295] [Archive] [answered]
This would allow me to export what to a "temporary" location so that I can review it.

**Comment by mdicss** [comment, marked as answer]
You already can use
`NEW_CLI=1 monaco download -e='environments.yaml' -s='environment' testdownload`
to save the download into a folder 'testdownload'.
I use this to be able to compare the new download to a older. But I miss a standard yaml/json structure, see: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/discussions/279

Maybe there should be a `-p` switch for the project in the commandline but it seems only to work without this in the new cli.
